src/library-authoring/data/apiHooks.ts

```diff
@@ -53,11 +53,10 @@ export const libraryAuthoringQueryKeys = {
     'libraryBlockTypes',
   ],
   xblockFields: (contentLibraryId: string, usageKey: string) => [
-    ...libraryAuthoringQueryKeys.all,
```

Comment:
@rpenido Why is this being deleted? If the query `libraryAuthoringQueryKeys.all` is invalidated, then `xblockFields` is not invalidated.

Reply:
Sorry. Not intended! This was fixed after I merged the last changes.